listenbrainz/db/dump_manager.py

```diff
@@ -111,14 +111,25 @@ def create_full(location, threads, dump_id, last_dump_id):
             write_hashes(dump_path)
         except IOError as e:
             current_app.logger.error('Unable to create hash files! Error: %s', str(e), exc_info=True)
-            return
+            sys.exit(-1)
 
+        try:
+            if not sanity_check_dumps(dump_path, 12):
+                return sys.exit(-1)
+        except IOError as e:
+            sys.exit(-1)
 
         # if in production, send an email to interested people for observability
         send_dump_creation_notification(dump_name, 'fullexport')
 
         current_app.logger.info('Dumps created and hashes written at %s' % dump_path)
 
+        # Write the DUMP_ID file so that the FTP sync scripts can be more robust
+        with open(os.path.join(dump_path, "DUMP_ID.txt"), "w") as f:
+            f.write("%s %s full\n" % (end_time.strftime('%Y%m%d-%H%M%S'), dump_id))
```

Comment:
`end_time.strftime('%Y%m%d-%H%M%S')` is done 3 times, perhaps store in a variable to reduce code redundancy.